april_vision/cli/live.py

```diff
@@ -84,10 +85,42 @@ def main(args: argparse.Namespace):
             except RuntimeError:
                 pass
 
+        for marker in markers:
+            output = "#{}, {}, ({}, {})".format(
+                marker.id,
+                marker.marker_type.value.strip('tag'),
+                int(marker.pixel_centre.x),
+                int(marker.pixel_centre.y),
+            )
+            # Check we have pose data
+            has_pose = True
+            try:
+                _ = marker.cartesian
+            except RuntimeError:
+                has_pose = False
+
+            if has_pose:
+                output += ", D={}, θ={}, φ={} | x={}, y={}, z={} | r={}, p={}, y={}".format(
+                    int(marker.distance),
+                    int(degrees(marker.spherical.theta)),
+                    int(degrees(marker.spherical.phi)),
+                    int(marker.cartesian.x),
+                    int(marker.cartesian.y),
+                    int(marker.cartesian.z),
+                    int(degrees(marker.orientation.roll)),
+                    int(degrees(marker.orientation.pitch)),
+                    int(degrees(marker.orientation.yaw)),
+                )
+            else:
+                output += ", no values for pose estimation"
+
+            LOGGER.info(output)
+
         cv2.imshow('image', frame.colour_frame)
 
         button = cv2.waitKey(1) & 0xFF
-        if button == ord('q'):
+        if (button == ord('q')) or (button == 27):
```

Comment:
This can simplify to `if button in (ord('q'), 23):`, though you might want to extract a constant to name the 27. (Named constants are usually clearer than comments)